src/main/java/org/folio/service/mapping/processor/translations/TranslationsHolder.java

```diff
@@ -126,16 +128,29 @@ public String apply(String updatedDate, int currentIndex, Translation translatio
   SET_FIXED_LENGTH_DATA_ELEMENTS() {
     private transient DateTimeFormatter originCreatedDateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
     private transient DateTimeFormatter targetCreatedDateFormatter = DateTimeFormatter.ofPattern("yyMMdd");
+    private String datesOfPublicationPattern = "datesOfPublication";
```

Comment:
these are constants, please use the proper naming, with capitals and final for more readability

Reply:
sure, I will push changes now